lib/initRedux.js

```diff
@@ -0,0 +1,32 @@
+import { createStore, applyMiddleware, combineReducers, compose } from 'redux';
+import thunkMiddleware from 'redux-thunk';
+import { composeWithDevTools } from 'redux-devtools-extension';
+import activeEvent from '../src/reducers/reducer_active_event';
+
+let reduxStore = null;
+
+function create(apollo, initialState = {}) {
+  return createStore(
+    combineReducers({
+      activeEvent: activeEvent,
+      apollo: apollo.reducer(),
+    }),
+    initialState,
+    compose(
+      applyMiddleware(apollo.middleware()),
+      composeWithDevTools(applyMiddleware(thunkMiddleware)),
+    )
+  )
+}
+
+export default function initRedux(apollo, initialState) {
```

Comment:
I need process.browser check to Make sure to create a new store for every server-side request so that data isn't shared between connections (which would be bad)
and reduxStore check to Reuse store on the client-side, that's why I use function instead of the object